app/src/main/res/layout-land/onboarding_slide.xml

```diff
@@ -46,24 +47,22 @@
         android:gravity="center_horizontal"
         android:text="@{viewModel.title}"
         android:textColor="@color/oppiaPrimaryText"
-        android:textSize="24sp"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="1.0"
         app:layout_constraintStart_toEndOf="@+id/slide_image_view"
         app:layout_constraintTop_toTopOf="parent" />
 
       <TextView
         android:id="@+id/slide_description_text_view"
+        style="@style/Subtitle1"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:layout_marginStart="20dp"
         android:layout_marginTop="16dp"
         android:layout_marginEnd="40dp"
-        android:fontFamily="sans-serif"
         android:gravity="center_horizontal"
         android:text="@{viewModel.description}"
         android:textColor="@color/oppiaPrimaryText"
```

Comment:
I think this color is part of Subtitle1, if so can you remove this attribute here?

Reply:
Done.